### ISSUE TITLE
Broken Links

### ISSUE THREAD

**Issue by ksuderman** [lapps/org.lappsgrid.example.java.whitespacetokenizer#16]
The Maven site that is deployed for Lappsgrid modules has changed so links to the Javadoc in the tutorial are broken...

**Comment by ksuderman** [comment]
I have some students working through the tutorial this week.  I will have them identify broken links and submit pull requests.

**Comment by keighrim** [comment]
Yes, this was on my todo list. I'll update the tutorial very very soon.

A quick sanity check. 
We've been updating example code at https://github.com/lapps/org.lappsgrid.examples . Are you sure the students working with the right tutorial?
